operation/manager.go

```diff
@@ -86,13 +87,19 @@ type IsPoweredInterface interface {
 }
 
 // WaitTillNotPowered waits until IsPowered returns false.
-func (sm *SingleOperationManager) WaitTillNotPowered(ctx context.Context, pollTime time.Duration, powered IsPoweredInterface) error {
+func (sm *SingleOperationManager) WaitTillNotPowered(ctx context.Context, pollTime time.Duration, powered IsPoweredInterface,
```

Comment:
I know you added some tests in the drivers, but add a test for this in manager_test too

Reply:
added